docs/getting_started.md

```diff
@@ -51,25 +47,46 @@ virt-install \
 ```
 
 Watch the OS console of the virtual machine to see the progress of the installation, waiting until the machine is rebooted and the login prompt appears.
+The OS console is also accessible from the `virt-manager GUI` as a result of running `sudo virt-manager`. 
 
 ## Access MicroShift
 
-Log into the hypervisor machine using your user credentials.
+From the OS console of the virtual machine, it is possible to log into the machine using your user credentials, `redhat:redhat`.
 
-Is it most convenient to access the MicroShift virtual machine using SSH. Run the following command to get the machine IP address and use it to remotely connect to the system.
+However, it is most convenient to access the MicroShift virtual machine using SSH.
+First, get the machine IP address with the following command.
 
 ```bash
 sudo virsh domifaddr microshift-starter
+#example output
+ Name       MAC address          Protocol     Address
+-------------------------------------------------------------------------------
+ vnet2      52:54:00:6d:08:f7    ipv4         192.168.122.2/24
+```
+
+Then, verify the virtual machine is accessable using SSH and the user credentials, `redhat:redhat`.
+
+```bash
+sudo ssh redhat@192.168.122.2 # when prompted, password is `redhat`.
+exit
 ```
 
-First, copy your pull secret file to the MicroShift virtual machine using `redhat:redhat` credentials.
+Copy your pull secret file to the MicroShift virtual machine using `redhat:redhat` credentials.
 
 ```bash
 USHIFT_IP=192.168.122.2
-scp ~/.pull-secret.json redhat@${USHIFT_IP}:
+sudo scp ~/.pull-secret.json redhat@${USHIFT_IP}:
 ```
 
-Log into the MicroShift virtual machine using `redhat:redhat` credentials. Run the following commands to configure `CRI-O` for using the pull secret and start the MicroShift service.
+Log into the MicroShift virtual machine.
+
+```bash
+sudo ssh redhat@192.168.122.2 # when prompted, password is `redhat`
```

Comment:
```suggestion
ssh redhat@${USHIFT_IP} # when prompted, password is `redhat`
```

Reply:
turns out I had too many auth failures w/ non-root ssh bc ssh was trying all my old rsa keys I had in there, once I cleared them out I get the password prompt :)

Reply:
updated to remove sudo & pushed from local, thanks